### PR TITLE
bitwise_not_

### LINKED ISSUE #17484: bitwise_not_

### PR DESCRIPTION
**Pull request by jakubbober**
Close #17484

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py [hunk 1]
```diff
@@ -6993,6 +6993,40 @@ def test_torch_instance_atan2_(
...
 
 
+# bitwise_not_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="bitwise_not_",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
+        num_arrays=2,
```
Comment (bipinKrishnan) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:7003]: Hey @jakubbober, is there a need for generating two arrays here?

Comment (jakubbober) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:7003]: Hi, I actually don't think so, but I wanted to match the logic from `test_torch_instance_bitwise_not_` (it also generates two arrays).

Comment (jakubbober) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:7003]: @bipinKrishnan should I change it or keep it consistent with the other function?

Comment (bipinKrishnan) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py:7003]: Okay, makes sense :slightly_smiling_face:

**Comment by bipinKrishnan** [comment]
LGTM! I will merge this PR now. Thanks for the contribution @jakubbober :rocket:

**Review by bipinKrishnan — APPROVED** [review]

**Pull request merged**